src/data/var/acceptData.js

```diff
@@ -13,7 +13,6 @@ return function( owner ) {
 	//    - Node.DOCUMENT_NODE
 	//  - Object
 	//    - Any
-	/* jshint -W018 */
```

Comment:
If no one would object, i will add this rule to `eslint-config-jquery`.

Thanks